ucm2/Meson/p241/p241.conf

```diff
@@ -0,0 +1,8 @@
+# Use case for the p241 Amlogic s805x reference design
```

Comment:
What is 'Meson' as the directory name? The top level directory should be a SoC maker name.

Reply:
Meson is name commonly used for Amlogic in the kernel - Like Sunxi for Allwinner
I'll change it to Amlogic